cosmos-gui/src/cmd_runner.js

```diff
@@ -28,18 +28,19 @@ var spawn = require('child_process').spawn;
 
 function run(cmd, params, callback) {
     var execution = spawn(cmd, params);
-    var result = '';
+    var resStdout = '';
+    var resStderr = '';
 
     execution.stdout.on('data', function (data) {
-        result += 'stdout: ' + data.toString();
+        resStdout += data.toString();
     });
 
     execution.stderr.on('data', function (data) {
-        result += 'stderr: ' + data.toString();
+        resStderr += + data.toString();
     });
 
     execution.on('close', function (code) {
-        return callback(null, result);
+        return callback(null, resStdout.length == 0 ? null : resStdout);
```

Comment:
Proposed change implemented in PR https://github.com/telefonicaid/fiware-cosmos/commit/e034c0f1a702d25551579593d0d1248e6d520e5c

Nevertheless, still trying to know why `resStderr` is always equals to `NaN`, even if the command execution goes well...